### ISSUE TITLE
My Vision Board overlapping images don't save as originally set up

### ISSUE THREAD

**Issue by jenkline** [ancaipen/Provia#351]
Jerome has noticed that when he creates vision boards with overlapping images, the saved vision board isn't displaying the images like they were originally set up. He said that the overlaps weren't in the order they originally were. See his screen shot attached. He doesn't have a screen shot of what the board looked like before he saved it, unfortunately. I wonder if this has anything to do with the issue Sean mentioned where the styling is not applied on the saved boards? 
![My Vision Board Screengrab](https://user-images.githubusercontent.com/104006203/165170421-037f22a2-003a-4ab4-b809-cac6ebd6a0ec.jpg)

**Comment by ancaipen** [comment]
@SeanIsAtWork - I'm looking into the layering issue.  Can you look into the css styles?  From what I can see the layering and css are 2 different issues.  It looks like this is only happening on-load of a vision board.

**Comment by SeanIsAtWork** [comment]
@ancaipen  I'll take a closer look into the css styles today, I delved into it a bit yesterday but had no luck,

**Comment by SeanIsAtWork** [comment]
@ancaipen I just updated the my projects CSS sheet to include my new styles for the vision board

**Comment by ancaipen** [comment]
@SeanIsAtWork - I made some adjustments around layering to force the same layers as they are dragged to the board.  Below is an example of what I dragged and saved.  I'm still not seeing the styles applied though when I selected a project after re-loading the pages.

![image](https://user-images.githubusercontent.com/43222107/165361507-9d5a38ad-2e7b-48fa-9352-62c434d25844.png)

![image](https://user-images.githubusercontent.com/43222107/165361590-01a6f596-92aa-48b7-ab1f-e084dca033f1.png)
